README.md

```diff
@@ -1,29 +1,40 @@
-# SOLID FORMS
+# Solid SDK Forms
 
 <img src="https://solid.inrupt.com/themes/custom/solid/logo.svg" width="150" height="150">
 
-Solid Form project to handle forms using ShEx and SHACL.
+This is a standalone Typescript library designed to handle form conversion and other form-related features.
```

Comment:
Maybe "This is a standalone Typescript library designed to handle the creation of Form Models from shapes, and other form-related features."?